lib/browser/rpc-server.js

```diff
@@ -234,6 +236,13 @@ ipcMain.on('ELECTRON_BROWSER_REQUIRE', function (event, module) {
   }
 })
 
+ipcMain.on('ELECTRON_BROWSER_READ_FILE', function (event, file) {
+  fs.readFile(file, (err, data) => {
+    if (err) event.returnValue = {err: err.message}
+    else event.returnValue = {data: data.toString()}
```

Comment:
Actually, the code that handles `Buffer` deserialization is in `lib/renderer/api/remote.js`, which is not used yet by sandboxed renderers. I will address this in another PR, as described by #6712